go/cmd/dolt/commands/init.go

```diff
@@ -132,7 +134,7 @@ func (cmd InitCmd) Exec(ctx context.Context, commandStr string, args []string, d
 		}
 	}
 
-	err := dEnv.InitRepoWithTime(context.Background(), types.Format_Default, name, email, initBranch, t)
+	err := dEnv.InitRepoWithTime(context.Background(), types.Format_Default, name, email, initBranch, t, apr.Contains(fun))
```

Comment:
Rather than introduce a new parameter to this method, you should introduce a new method that does this specific thing

Reply:
Ah, that makes sense! Wouldn't this be unnecessary if it's all done at the application/command layer though?

Reply:
Yeah, I gave this feedback before reconsidering the overall approach.